test-framework/junit5/src/main/java/io/quarkus/test/junit/QuarkusIntegrationTestExtension.java

```diff
@@ -214,7 +214,7 @@ private void injectTestContext(Object testInstance) {
         Class<?> c = testInstance.getClass();
         while (c != Object.class) {
             for (Field f : c.getDeclaredFields()) {
-                if (f.getType().equals(QuarkusIntegrationTest.Context.class)) {
+                if (f.getType().equals(DevServicesContext.class)) {
```

Comment:
Hi @rsvoboda I wonder if updating `QuarkusIntegrationTestExtension` is too early, it is fine as it is, but `DevServicesContext` has only been introduced in `main` so removing the ability to the support the injection of `QuarkusIntegrationTest.Context` is early, not sure how much it is used but I can imagine the users seeing theor integration tests starting failing with `2.3.x`, just occurred to me this morning...
I think adding a note that it is deprecated to the 2.3 migration guide is what is needed at this stage